### ISSUE TITLE
[suggestion] integrate lunr.js for in-page search

### ISSUE THREAD

**Issue by seralf** [okfn/handbook#81]
I like to suggest the testing of a small integration, while developing the new theme.
As long as the content grows it could be good to have some onsite search, such as the services provided by Solr / Elasticsearch . A good lightweight alternative that works on the client side (thus on a static site, for example, once rendered by jekyll), could be [Lunr.js](http://lunrjs.com/). It should be easy to integrate, and interesting to test, for ehancing contents, especially in case of multi-page internally sub-divided, or very long pages.

**Comment by danfowler** [comment]
Sounds great :).  Would you be up for adding?

**Comment by seralf** [comment]
of course! the only problem for me is that my contributions in this phase are rather slow, due to lack of time, but on the other hand I suppose there is not any particular deadline :-)
